config/default.json

```diff
@@ -34,20 +34,20 @@
   },
   "provider": {
     "source": {
-      "type": "NFS",
+      "kind": "NFS",
       "pvPath": "/Path/To/Models"
     },
     "dest": {
-      "type": "S3",
-      "accessKeyId": "minioadmin",
-      "secretAccessKey": "minioadmin",
-      "endpointUrl": "http://127.0.0.1:9000",
-      "bucket": "3dtiles",
-      "region": "us-east-1",
+      "kind": "s3",
```

Comment:
What did you temove the SSL?

Reply:
don't need it as we provide https:// in the full URl